libraries/ESP8266WiFi/src/BearSSLHelpers.h

```diff
@@ -133,17 +133,57 @@ class X509List {
 // significantly faster.  Completely optional.
 class WiFiClientSecure;
 
+// Cache for a TLS session with a server
+// Use with BearSSL::WiFiClientSecure::setSession
+// to accelerate the TLS handshake
 class Session {
   friend class WiFiClientSecureCtx;
 
   public:
     Session() { memset(&_session, 0, sizeof(_session)); }
   private:
     br_ssl_session_parameters *getSession() { return &_session; }
-    // The actual BearSSL ession information
+    // The actual BearSSL session information
     br_ssl_session_parameters _session;
 };
 
+// Cache for the TLS sessions of multiple clients.
+// Use with BearSSL::WiFiServerSecure::setCache
+class ServerSessions {
+  friend class WiFiClientSecureCtx;
+
+  public:
+    // Uses the given buffer of the given size to store the size and initializes it.
+    ServerSessions(uint8_t *store, uint32_t size) : ServerSessions(store, size, false) {}
```

Comment:
Same comment about size->sessioncount here...

Reply:
In order to implement this, I had to create a type with a size of 100 bytes called `ServerSession`.  
If someone wants to use this method, they can simply create an array of `ServerSession` and pass it to this constructor.